charts/rpe-send-letter-service/values.yaml

```diff
@@ -1,37 +1,58 @@
 java:
   applicationPort: 8485
+  aadIdentityName: send-letter
   keyVaults:
     rpe-send-letter:
-      resourceGroup: rpe-send-letter-service
       secrets:
         - ftp-user
         - ftp-private-key
         - ftp-public-key
+        - upload-summary-report-recipients
+        - send-letter-service-POSTGRES-PASS
+        - encryption-public-key
+        - app-insights-instrumentation-key
   environment:
     # db
-    LETTER_TRACKING_DB_HOST: "{{ .Release.Name }}-postgresql"
-    LETTER_TRACKING_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
-    LETTER_TRACKING_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
-    LETTER_TRACKING_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    LETTER_TRACKING_DB_HOST: "rpe-send-letter-service-db-{{ .Values.global.environment }}.postgres.database.azure.com"
+    LETTER_TRACKING_DB_PORT: "5432"
+    LETTER_TRACKING_DB_NAME: "send_letter"
+    LETTER_TRACKING_DB_USER_NAME: "send_letter@rpe-send-letter-service-db-{{ .Values.global.environment }}"
+    LETTER_TRACKING_DB_CONN_OPTIONS: "?sslmode=require"
+
     # db - migration
-    FLYWAY_URL: "jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/{{ .Values.postgresql.postgresqlDatabase}}"
-    FLYWAY_NOOP_STRATEGY: "false"
+    FLYWAY_URL: "jdbc:postgresql://rpe-send-letter-service-db-{{ .Values.global.environment }}.postgres.database.azure.com:5432/send_letter?sslmode=require"
+
     # encryption
-    ENCRYPTION_ENABLED: "false"
-    ENCRYPTION_PUBLIC_KEY: ""
+    ENCRYPTION_ENABLED: "true"
```

Comment:
wont work without a key

Reply:
removed. it is in bootstrap.

Reply:
why is it enabled now?

Reply:
encryption is and should remain "on". for previews it's ok to have disabled. such situation **is** represented in this pr